web/pgadmin/dashboard/static/js/SystemStats/Utility.jsx

```diff
@@ -0,0 +1,85 @@
+import url_for from 'sources/url_for';
+import { DATA_POINT_SIZE } from 'sources/chartjs';
+
+export const X_AXIS_LENGTH = 75;
+
+/* URL for fetching graphs data */
+export function getStatsUrl(sid=-1, did=-1, chart_names=[]) {
```

Comment:
I don't see any react code in this file. Why is it named like a component. It should be something like `utility.js` or `common.js`